sheet/CommonTypes.qml

```diff
@@ -0,0 +1,22 @@
+pragma Singleton
+import QtQuick 2.0
+
+import org.lasath.psychic_bear 1.0
+
+Item {
+    property BonusType armour: BonusType {
+        stacking: false
+    }
+
+    property BonusType acDex: BonusType {
+        stacking: false
+    }
+
+    property BonusType sheild: BonusType {
```

Comment:
I'm pretty sure property names have to start with a lowercase letter.
If these names ever need to be printed, we'll probably need to add a 'name' property to BonusType.